### PR TITLE
May run pydantic failure fixes

### PR DESCRIPTION
**Pull request by sierra-moxon**
- gtopdb, signor, chembl failed last ingest as a result of more strict pydantic validation in linkml upgrade we did.  unfortunately our test suite did not catch these errors (it did catch errors for a few other ingests that were fixed ahead of this PR when linkml was updated.  Look for another PR that does a comprehensive "test data" suite that exercises each edge declared in the rigs for passage so that if we upgrade biolink or linkml in this repo, we can test that upgrade ahead of PROD deployment with confidence).

- this also involved some small changes to biolink associations.  I did not change any existing edges in any of these three ingests (except flipping the direction of _some_ "affected_by" edges in chembl ingest to "affects" edges.  the subject/object qualfiiers were also swapped) so that we wouldn't have to worry that edges were changing without proper QA/QC.  

- The associations in biolink have the same edge properties/qualifiers available to them as the old associations in chembl did, just different predicate constraints (e.g. has_substrate).  This was also caught by Matt in the RIG for chembl, but not addressed yet.  Now with sticter validation, these changes had to go in.  The rig was also adjusted here. 

to test this, I:
- ran the test suite locally, all passed
- ran all the ingests in our repo locally because of the biolink-model changes.  All pass without error.  I did notice the new reporting for NodeNorm failures, and there are several ingests that need tickets put into babel to help; that will be a separate effort here. I like the new reporting!

### CODE REVIEW

File: src/translator_ingest/ingests/signor/signor.py [hunk 1]
```diff
@@ -546,7 +544,7 @@ def transform_ingest_all(koza: koza.KozaTransform, data: Iterable[dict[str, Any]
                     agent_type=AgentTypeEnum.manual_agent,
                     ## five edge attributes in order
                     predicate = current_predicate_mapping,
-                    qualified_predicate = BIOLINK_AFFECTS,
+                    qualified_predicate = BIOLINK_CAUSES,
```
Comment (sierra-moxon) [inline src/translator_ingest/ingests/signor/signor.py:547]: note this is a change, but it matches the pattern of `predicate`, `affects`, `qualified_predicate`, `causes` that we are familiar with.

File: src/translator_ingest/ingests/chembl/chembl_qualifiers.json [hunk 2]
```diff
@@ -192,7 +192,7 @@
         }
     },
     "OTHER": {
-        "association": "ChemicalAffectsGeneAssociation",
+        "association": "ChemicalGeneInteractionAssociation",
         "predicate": "biolink:interacts_with",
         "qualifiers": {}
     },
```
[the remark this answers is not in the document]
Comment (sierra-moxon) [reply src/translator_ingest/ingests/chembl/chembl_qualifiers.json:197]: removed.

**Review by bazarkua — APPROVED** [review]
